resources/markdown/who_audit_l2.md

```diff
@@ -0,0 +1,282 @@
+
+### Inclusion:
+<pre>
+Patient is >= 18 years old
+AND most recent Alcohol PS Question response is 'YES'
+AND
+     Most recent (WHO) AUDIT-C Question #1 response is 'Never'
+OR
+     Most recent (WHO) AUDIT-C Question #1 score < 3
+     AND most recent (WHO) AUDIT-C Question #2 score = 0
+     AND most recent (WHO) AUDIT-C Question #3 score = 0
+     AND
+           Female
+          OR Sex at birth is unknown
+          Male > 65 years old 
+ OR   
+      Most recent (WHO) AUDIT-C Question #1 score < 4
+      AND most recent (WHO) AUDIT-C Question #2 score = 0
+      AND most recent (WHO) AUDIT-C Question #3 score = 0
+      AND Male
+</pre>
+
+### Exclusion:
+<pre>
+Carry over exclusions from Step 1
+</pre>
+
+### Interventions and Actions:
+<pre>
+- Display Question #9 & Question #10 of the (WHO) AUDIT
+- Record and display responses to Question #9 & Question #10  of the (WHO) AUDIT
+</pre>
+
+--------------------------------------------------------------------------------------------------------------------------------------------
+# STEP 5: Consider additional screening: 
+
+## LOGIC PATH #1: Provide full (WHO) AUDIT:
+
+### Inclusion:
+<pre>
+Patient is >= 18 years old
+AND most recent Alcohol PS Question Response is 'YES'
+AND NOT most recent (WHO) AUDIT-C Q1 response is 'Never'
+AND most recent (WHO) AUDIT-C score >=3
+     AND Female
+     OR Male > 65 years old
+     OR Sex at birth is unknown
+OR most recent (WHO) AUDIT-C score >=4 AND Male <=65 years old
+</pre>
+
+### Exclusion:
+<pre>
+Carry over exclusions from Step 1
+</pre>
+
+### Interventions and Actions:
+<pre>
+- Display Question #4-Question #10 of the (WHO) AUDIT
+- Record and display responses to Question #4-Question #10 of the (WHO) AUDIT
+- Calculate score of (WHO) AUDIT
+- Record score of (WHO) AUDIT
+</pre>
+
+--------------------------------------------------------------------------------------------------------------------------------------------
+# STEP 6: Consider pregnancy status: 
+
+## LOGIC PATH #1: Post (WHO) AUDIT Questions #1, #9, and #10:
+
+### Inclusion:
+<pre>
+Patient is >= 18 years old and < 50 years old
+AND Female
+AND most recent (WHO) AUDIT-C Question #1 response is 'Never'
+AND most recent response to (WHO) AUDIT Question #9 and #10 is present
+</pre>
+
+### Exclusion:
+<pre>
+Carry over exclusions from Step 1
+AND Pregnant (active, recurring)
+    OR Pregnancy Observation in past 42 weeks (final, amended, corrected)
+    OR Hysterectomy (completed)
+</pre>
+
+### Interventions and Actions:
+<pre>
+- Display 'Pregnancy Question'  (i.e., Are you pregnant or trying to become pregnant?)
+- Record and display 'Pregnancy Question' response
+</pre>
+
+----------------------------------------------------------------------
+## LOGIC PATH #2: Post (WHO) AUDIT-C (with or without responses to Question #9 and Question #10):
+
+### Inclusion:
+<pre>
+Patient is >= 18 years old and < 50 years old
```

Comment:
Dave: any concerns with this "sum" logic phrase? There are so many combinations of responses that could add up to <3, that the human readable logic would be super long. Can we stay at this high level for the L2 and let there be more math under the hood to create the L3 version? If not, please provide me with the proposed logic

Reply:
SS: After group discussion, the "sum" logic phrase was replaced by what is posted to the left. Additionally, Logic paths #1 and #2 were combined so that all patients who score  "negative" for the (WHO) AUDIT-C  will be presented with the pregnancy question after they finish screening. -